### ISSUE TITLE
[Problem/Bug]: WebView creation fails when app is started using "Run as administrator" and explicit admin credentials

### ISSUE THREAD

**Issue by Urmeli0815** [MicrosoftEdge/WebView2Feedback#4672]
### What happened?

I have the following scenario: my machine has two accounts
- "michael" who is a local admin
- "basil" who is a standard user

My app was installed by "michael" for all users using elevated rights.

Now "basil" logs in. He rights-click on our app-icon and selects "Run as administrator". He now enters the credentials from "michael". The app start fails with this error message from WebView2:

![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/7428276/b14d8b0b-92d0-4403-9376-84c55d8ffd96)

We set the UserDataFolder of WebView2 to a LocalAppData-Path of the running user (which in this case is "michael"). But it seems that child processes of WebView2 don't use the same user. In fact they run under the "basil" user as can be seen here:

![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/7428276/0237d486-b282-49e5-a109-a4b25a04a0bb)

My expectation would be that the child processes get launched with the same user as the app that contains the WebView2.

Can you clarify if this is expected behavior or if this is a bug?

### Importance

Moderate. My app's user experience is affected, but still usable.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

126.0.2592.87

### SDK Version

1.0.1518.46

### Framework

Win32

### Operating System

Windows 10, Windows 11

### OS Version

10.0.26120

### Repro steps

The problem should be reproducible with any app using WebView2. It just depends on the described admin/non-admin user scenario.

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

No, this never worked

**Comment by Urmeli0815** [comment]
Probably the same problem as in https://github.com/MicrosoftEdge/WebView2Feedback/issues/3128 but I decided to create a new issue with a more detailed description.

**Comment by VishyBala007** [comment]
Hi All, 

Any update what i found is when the app is running in elevated mode, the webview2 user data folder is always created in the executing user Local state folder. not the logged in user.


So the logged in user who is not an admin is not able to access the local state of the Admin user..

this is because webview2 runtime exe is already running in non admin account, though our winui3 app is running in Admin Account

**Comment by Urmeli0815** [comment]
I'm aware that MS closed other issues for the same topic as unresolved. But for me and other devs it's still a topic. We regularly get feedback from customers who run into this situation. One customer solved it by allowing the low-privileged user access to the folder of the high-privileged user; but understandably this concerns the customer.

You should come up with a solution for that.

In my view it should be the responsibility of the developer to decide if the WebView runs as the same user or not. In our case we serve the web resources locally, so we have complete control of what gets into the WebView.

**Comment by VishyBala007** [comment]
As per my interaction with Microsoft on this ticket.

Solution:

In case of elevated scenario change the UDF folder .. So that all users can access the path.

best place Programdata folder

```
var environment = await CoreWebView2Environment.CreateWithOptionsAsync(null, "CustomDataFolderPath", null);
await MyWebView.EnsureCoreWebView2Async(environment);
```

**Comment by Urmeli0815** [comment]
@VishyBala007 Yes, that could be a solution. But I have some security concerns about it.
